README.md

```diff
@@ -113,9 +114,11 @@ This sets up the port forwarding and passes the .env file through to set up the
 
 You should then be able to see the application as usual at 127.0.0.1:5000
 You will need to make sure you create .env as a file to put your environment variables in, this needs to include:
-API_KEY=XXX
-SERVER_TOKEN=XXX
-BOARD_ID=XXX
+MONGO_CONNECTION=mongodb+srv
+MONGO_SRV=XXX
+MONGO_DB=XXX
+MONGO_USER=XXX
+MONGO_PWD=XXX
```

Comment:
Minor, but it might be easier to simply ask the user for a connection string, and let them worry about precisely how they compose this